mail-mta/postfix/postfix-3.4.5.ebuild

```diff
@@ -60,10 +60,11 @@ REQUIRED_USE="ldap-bind? ( ldap sasl )"
 
 S="${WORKDIR}/${MY_SRC}"
 
+PATCHES=( "${FILESDIR}/${PN}-libressl-certkey.patch"
+	"${FILESDIR}/${PN}-libressl-server.patch"
+)
```

Comment:
please use proper indenting:
```
PATCHES=(
	"${FILESDIR}/${PN}-libressl-certkey.patch"
	"${FILESDIR}/${PN}-libressl-server.patch"
)
```

Reply:
This is copy-pasted this from postfix-3.5_pre20190330-r1.ebuild... It's fixed now and I pushed [a PR for 3.5 too](https://github.com/gentoo/gentoo/pull/11853).